### PR TITLE
adapt building info panel to meet specs

### LINKED ISSUE #568: Update the detailed information panel
[Link to the specifications](https://www.figma.com/file/dUL9ybnag2LhQpicJseFaH/Seismic-Risk?node-id=1637%3A289)
(the value for `Clasa de risc` should be U3, the `risk_category` API variable while the `Anul expertizei tehnice` should be the value of the `examination_year` variable)

Screenshot 2021-02-26 at 10 32 38

### PR DESCRIPTION
**Pull request by myshy93**
### What does it fix?

Adapt building info panel to meet specifications,

Closes #568 

### How has it been tested?
**Manually on Google Chrome**

*Desktop version*
image
image

*Mobile Version*

image
image

### CODE REVIEW

File: client/src/components/BuildingDetails.js [hunk 1]
```diff
@@ -16,52 +16,65 @@ export default function BuildingDetails(props) {
...
+            <Col lg={{ span: 13, offset: 2 }} span={13}>
               <Trans>Address</Trans>:
-            </strong>
-            {details.address}
-          </p>
-          <p>
-            <strong>
+            </Col>
+            <Col lg={9} span={11}>
+              {details.address}
```
Comment (tudoramariei) [inline client/src/components/BuildingDetails.js:35]: The building address is composed of the street name and the street number
The `postcode` actually contains the street number information.

Comment (myshy93) [reply client/src/components/BuildingDetails.js:35]: So `{details.address} {details.postcode}`?

Comment (tudoramariei) [reply client/src/components/BuildingDetails.js:35]: Almost; it's more like this:
```
{details.address} no. {details.post_code}  // for English
{details.address} nr. {details.post_code}  // for Romanian
```

I know it's confusing to use the `post_code` for this but we'll get around and change the model & API to eliminate the confusion.

Comment (myshy93) [reply client/src/components/BuildingDetails.js:35]: any formatting for `postcode`? trim leading zeros?

Comment (tudoramariei) [reply client/src/components/BuildingDetails.js:35]: Yes, I think you can do that for good measure 🤔

File: client/src/components/BuildingDetails.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default function BuildingDetails(props) {
               <Trans>Address</Trans>:
             </Col>
             <Col lg={9} span={11}>
-              {details.address}
+              {details.address} <Trans>no.</Trans> {parseInt(details.post_code, 10)}
```
Comment (tudoramariei) [inline client/src/components/BuildingDetails.js:35]: I agree to trim leading zeros but we can't convert the street number into an `int`.
We have either one entry for multiple numbers (ex. 19-21) or building numbers with a letter (ex. 42B)

Comment (myshy93) [reply client/src/components/BuildingDetails.js]: removed parseInt

**Review by tudoramariei — APPROVED** [review]

**Pull request merged**